src/App/Fossa/Report/Attribution.hs

```diff
@@ -0,0 +1,141 @@
+module App.Fossa.Report.Attribution
+  ( Attribution (..),
+    Dependency (..),
+    License (..),
+    LicenseContents (..),
+    LicenseName (..),
+    Project (..),
+  )
+where
+
+import Prologue
+
+newtype LicenseName
+  = LicenseName {rawName :: Text}
+  deriving (Eq, Ord, Show, FromJSON, FromJSONKey, ToJSON, ToJSONKey)
```

Comment:
I wouldn't have thought of this -- this is good. nit: is `ToJSON` and `FromJSON` required here?

Reply:
Yeah it's used in values in other places.